lib/mpl_toolkits/tests/test_axes_grid1.py

```diff
@@ -409,8 +409,9 @@ def test_picking_callbacks_overlap(big_on_axes, small_on_axes, click_on):
     # In each case we expect that both rectangles are picked if we click on the
     # small one and only the big one is picked if we click on the big one.
     # Also tests picking on normal axes ("gca") as a control.
-    big = plt.Rectangle((0.25, 0.25), 0.5, 0.5, picker=5)
-    small = plt.Rectangle((0.4, 0.4), 0.2, 0.2, facecolor="r", picker=5)
+    big = plt.Rectangle((0.25, 0.25), 0.5, 0.5, picker=True, pickradius=5)
```

Comment:
Maybe you want to keep one of these for the test coverage?

Reply:
sure